reviewrot-build-template.yaml

```diff
@@ -0,0 +1,75 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: code-review-reminder-build-template
+  annotations:
+    openshift.io/display-name: "Code review reminder build"
+    description: "Description"
+    tags: "reviewrot code-review-reminder build"
+message: |
+  Code review reminder has been initialized.
+  Please add a certificate and add a modified config file
+  .reviewrot.yaml into secrets.
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    annotations:
+      openshift.io/display-name: "reviewrot"
+    name: reviewrot
+  spec:
+    tags:
+    - name: latest
+      annotation: null
+      from:
+        kind: DockerImage
+        name: quay.io/pbortlov/reviewrot:latest
+      importPolicy:
+        scheduled: true
+
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: ${APP}-build
+    labels:
+      app: build
+      service: ${APP}
+  spec:
+    source:
+      type: Git
+      git:
+        uri: ${GIT_URL}
+        ref: ${REF}
+    # Maybe change the strategy to get image from docker-registry
+    strategy:
+      type: Docker
+      dockerStrategy:
+        from:
+          name: registry.access.redhat.com/rhel7.5:latest
+          kind: DockerImage
+    output:
+      pushSecret:
+        name: pbortlov-reviewrot-bot-secret # ADD your secrets for quay bot
```

Comment:
Now everyone who wants to build this has to have a secret named after you. Could you please change it to something more generic? Or a parameter?

Reply:
yes